.github/workflows/main.yml

```diff
@@ -23,6 +23,13 @@ jobs:
       - name: Run clippy
         run: cargo clippy --all -- -Dwarnings
 
+      - name: Run compute-static tests
+        working-directory: terragrunt/modules/crates-io/compute-static
+        run: |
+          curl -LsSf https://get.nexte.st/latest/linux | tar zxf - -C ${CARGO_HOME:-~/.cargo}/bin
```

Comment:
I would recommend using `taiki-e/install-action` for this: 
```
- uses: taiki-e/install-action@v2
  with:
    tool: cargo-nextest,viceroy
```

Or directly `cargo binstall`

Reply:
Right, didn't know about this action, thank you!